status.json

```diff
@@ -5081,5 +5081,21 @@
       "priority": "Low"
     },
     "uservoiceid": 19594159
+  },
+  {
+    "name": "Media Queries: prefers-reduced-motion",
+    "category": "Accessibility",
+    "link": "https://drafts.csswg.org/mediaqueries-5/#prefers-reduced-motion",
+    "summary": "Used to detect if the user has requested the system minimize the amount of animation or motion it uses.",
+    "standardStatus": "Editor’s draft",
+    "ieStatus": {
+      "text": "Under Consideration",
+      "priority": "Low"
```

Comment:
![Higher, Higher. Girl on a swing](http://3.bp.blogspot.com/-YVNaHyNlM28/Tw9JoyBv3HI/AAAAAAAAALE/LwLDLPhtJ2U/s200/higher.jpg)

Reply:
@nschonni :D please upvote in UserVoice if you'd like and haven't already! https://wpdev.uservoice.com/forums/257854-microsoft-edge-developer/suggestions/19291531-add-support-for-css-prefers-reduced-motion-media-f

Reply:
Done! Posted on the web-a11y Slack to try and gets some more votes too